apps/challenges/views.py

```diff
@@ -42,6 +42,12 @@ def challenge_list(request, challenge_host_team_pk):
         return paginator.get_paginated_response(response_data)
 
     elif request.method == 'POST':
+
+        if not ChallengeHost.objects.filter(user=request.user).filter(team_name=challenge_host_team_pk):
```

Comment:
i think this has another problem too .I think it doesn't check for the condition in which the creator is there. I mean creator doesn't automatically get added to the team in `challengehost`. it is only in challenge table. creator will have an object in `challenge` only . so i should check both right ? @TroJan